### PR TITLE
View state estimation balance results (quality, status, quality criterion, quality per region)

### PR DESCRIPTION
**Pull request by FranckLecuyer**

### CODE REVIEW

File: src/redux/actions.ts [hunk 1]
```diff
@@ -1251,3 +1253,19 @@ export const addSortForNewSpreadsheet = (
...
+export function setStateEstimationResultFilter(
+    filterTab: keyof AppState[typeof STATEESTIMATION_RESULT_STORE_FIELD],
+    stateEstimationResultFilter: MutableUnknownArray
+): StateEstimationResultFilterAction {
+    return {
+        type: STATEESTIMATION_RESULT_FILTER,
+        filterTab: filterTab,
+        [STATEESTIMATION_RESULT_STORE_FIELD]: stateEstimationResultFilter,
+    };
+}
```
Comment (dbraquart) [inline src/redux/actions.ts:1271]: if it does not work and is not requested by PO, maybe we should rollback everything dealing with sort/filter in redux ?

File: src/components/results/stateestimation/state-estimation-status-result.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+                            <FormattedMessage id="StateEstimationQuality" />
+                        </Typography>
+                        <Typography sx={styles.valueTypography}>{result.qualityLevel}</Typography>
+                    </Stack>
+                )}
+            </>
+        );
+    };
+
+    return <>{renderStateEstimationStatusResult()}</>;
```
Comment (dbraquart) [inline src/components/results/stateestimation/state-estimation-status-result.tsx:49]: we could directly return the JSX code.

File: src/components/results/stateestimation/state-estimation-quality-result.tsx [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+    const intl = useIntl();
+    const gridRef = useRef<AgGridReact>(null);
+
+    const tableNameFormatted = intl.formatMessage({ id: tableName });
+
+    const stateEstimationStatus = useSelector(
+        (state: AppState) => state.computingStatus[ComputingType.STATE_ESTIMATION]
+    );
+
+    const [isQualityReady, setIsQualityReady] = useState(false);
```
Comment (dbraquart) [inline src/components/results/stateestimation/state-estimation-quality-result.tsx:44]: I dont understand this state. Seems to be always false ?

**Review by dbraquart — CHANGES_REQUESTED** [review]
code: What to do with sort/filter if it does not work ? clean the code, fix later ?)
tests: ok (except sort/filter)

File: src/components/results/stateestimation/state-estimation-status-result.tsx (hunk 2 above)
Comment (FranckLecuyer) [reply src/components/results/stateestimation/state-estimation-status-result.tsx:49]: Done

File: src/redux/actions.ts (hunk 1 above)
Comment (FranckLecuyer) [reply src/redux/actions.ts:1271]: Sorting and filtering removed

**Review by dbraquart — APPROVED** [review]
code: ok

tests: ok

**Pull request merged**